src/PowerModels.jl

```diff
@@ -1,17 +1,14 @@
 module PowerModels
 
-import LinearAlgebra, SparseArrays
-
+import InfrastructureModels
+import InfrastructureModels as _IM
```

Comment:
Wondering why we have both of these lines.  Isn't one sufficient?

Reply:
There was a test that accessed the full `InfrastructureModels` symbol:
https://github.com/lanl-ansi/PowerModels.jl/blob/95e6c2cb54e2fbe9b9c05c7e9e2618fe117e6d0b/test/matpower.jl#L203

Ideally, you wouldn't do this, and we could remove the first line, which would make it more obvious that the `_IM` symbol is private.